tests/conftest.py

```diff
@@ -322,12 +326,10 @@ def active_config(stub_config):
     """Loads the standard config as the active configuration returned by
     dallinger.config.get_config() and returns it.
     """
-    from dallinger.config import get_config
+    from dallinger import config as c
```

Comment:
This is weird, I don't remember making this change!

Reply:
Oh, @jessesnyder made that change on the original branch. Now it all makes sense.